deny.toml

```diff
@@ -0,0 +1,37 @@
+# https://embarkstudios.github.io/cargo-deny/
+
+targets = [
+    { triple = "aarch64-apple-darwin" },
+    { triple = "aarch64-linux-android" },
+    { triple = "x86_64-apple-darwin" },
+    { triple = "x86_64-pc-windows-msvc" },
+    { triple = "x86_64-unknown-linux-gnu" },
+    { triple = "x86_64-unknown-linux-musl" },
+]
+
+
+[advisories]
+vulnerability = "deny"
+unmaintained = "warn"
```

Comment:
Will these show up as a warning on the Action CI step as well?

Reply:
Yes, unmaintained crates will show up as warnings (but not as errors).
You can also run `cargo deny check` manually